src/components/schedule-visualizer/tests/places-overlay.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import officeMap from '../../../mock/data/building-map-office';
+import L from 'leaflet';
+import { Map as LMap } from 'react-leaflet';
+import PlacesOverlay from '../places-overlay';
+import Place from '../place';
```

Comment:
Places is not used in RMF anymore and is removed from in a recent PR, you can remove this test as well.

Reply:
Done, thanks!